tests/test_benchmark.py

```diff
@@ -252,11 +289,30 @@ def benchmark_dhfr(verbose=False, num_batches=100, steps_per_batch=1000):
     )
 
 
+def prepare_hif2a_initial_states(st, host_system, host_coords, host_box):
```

Comment:
nit: should the name of this function be singular?

Reply:
willfix